compiler/Makefile

```diff
@@ -4,9 +4,10 @@
 OCAMLC=ocamlc
 OCAMLLEX=ocamllex
 OCAMLYACC=ocamlyacc
-OBJS=parser.cmo scanner.cmo
+OBJS=parser.cmo scanner.cmo generator.cmo
 
-default: $(OBJS)
```

Comment:
Can we keep the `default` target for consistency with the other makefiles? Maybe

```makefile
default: generator

generator: $(OBJS)
    ocamlc -g -o generator $(OBJS)
```